docs/guide/basic-operations.md

```diff
@@ -336,11 +336,11 @@ try {
 ## isItPossibleTo* methods
 
 There are also methods that you may find useful to call in pair with
-the above mentioned operations. These methods are prefixed with
+the above-mentioned operations. These methods are prefixed with
 `isItPossibleTo*` whose sole purpose is to check if the desired
 operation is possible. They all return a simple `boolean` value.
 You will find it handy when you want to give the user a more generic
-message and you don't want to react to specific errors.
+message, and you don't want to react to specific errors.
```

Comment:
Just a small thing, I think comma isn't necessary here, because you aren't joining two separate statements (I think the indicator is that when you use `and` to split and create 2 separate sentences, they do not make sense on their own), you're just listing cases when it comes handy to use isItPossibleTo* methods.

Reply:
You're right. Fixed.